comps/cores/mega/orchestrator.py

```diff
@@ -40,7 +40,8 @@ def flow_to(self, from_service, to_service):
     async def schedule(self, initial_inputs: Dict, llm_parameters: LLMParams = LLMParams()):
         result_dict = {}
 
-        async with aiohttp.ClientSession(trust_env=True) as session:
+        timeout = aiohttp.ClientTimeout(total=1000)
```

Comment:
is timeout 1000 still not enough? do we need to expose the parameter through env variable when the image is available?

Reply:
1000 seconds is sufficient even for high concurrency benchmark tests. I think we should keep this as internal information. It will confuse users if we export it as an environment variable.